timeboost/src/binaries/timeboost.rs

```diff
@@ -100,13 +95,13 @@ async fn main() -> Result<()> {
         .iter()
         .map(|peer| {
             let sig_key = multisig::PublicKey::try_from(peer.sigKey.as_ref())
-                .expect("Failed to parse sigKey");
+                .expect("Should parse sigKey bytes");
```

Comment:
Since this is all external data I would use an error here (and `?`, potentially augmented with an anyhow context). The final `.collect::<Result<Vec<_>>>()?;` would propagate it.